examples/src/main/java/org/apache/spark/examples/ml/JavaOneVsRestExample.java

```diff
@@ -75,7 +75,7 @@ public static void main(String[] args) {
 
     // compute the classification error on test data.
     double accuracy = evaluator.evaluate(predictions);
-    System.out.println("Test Error : " + (1 - accuracy));
+    System.out.println("Test Error = " + (1 - accuracy));
```

Comment:
Some of these changes feel kind of trivial, but I guess this is for consistency. But other new System.out.println statements use : not =